package/base-files/Makefile

```diff
@@ -199,7 +199,8 @@ define Package/base-files/install
 		$(1)/usr/lib \
 		$(1)/usr/bin \
 		$(1)/sys \
-		$(1)/www \
+		$(1)/www
+	mkdir -p -m 750 \
```

Comment:
This change looks unrelated, please drop it from this PR.

Reply:
Moved to PR #13892.